### PR TITLE
foldit: Add version nightly

### PR DESCRIPTION
**Pull request by schance995**
<https://fold.it/portal/info/about>

"Foldit is a revolutionary crowdsourcing computer game enabling *you* to contribute to important scientific research. "

Although this is a game I think it belongs in the extras bucket because it appeals to a wider audience.

Also the installer on their website is not the latest, running it the first time automatically updates it. That's why I put a version number of 1.

### CODE REVIEW

File: bucket/foldit.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "description": "protein folding game for scientific research",
+    "license":  {
+        "identifier": "Freeware",
+        "url": "https://fold.it/portal/legal"
+    },
+    "url":  "https://files.ipd.uw.edu/pub/foldit/Foldit-win_x86.exe#dl.7z",
+    "homepage":  "https://fold.it/portal/",
+    "hash":  "f92e30c8dc0c1c7502940206d095172fba9676104a16693595266835cd798456",
+    "bin":  "Foldit.exe",
+    "post_install": "Remove-Item \"$dir\\uninstall.exe\"",
```
Comment (Ash258) [inline bucket/foldit.json:12]:
```suggestion
    "version": "nightly",
    "description": "Crowdsourcing computer game contributing to important scientific research",
    "homepage": "https://fold.it",
    "license": {
        "identifier": "Freeware",
        "url": "https://fold.it/portal/legal"
    },
    "url": "https://files.ipd.uw.edu/pub/foldit/Foldit-win_x86.exe#/dl.7z",
    "pre_install": "Remove-Item \"$dir\\`$*\", \"$dir\\uninst*\" -Recurse",
    "bin": "Foldit.exe",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Review by Ash258 — CHANGES_REQUESTED** [review]
Find a way how to manually persist all themes, configs, GPUs, and cmp-* folders

**Comment by schance995** [comment]
This is my persist list so far:
```
"persist": [
    "cmp-binary-00000000000000000000000000000000/",
    "cmp-resources-00000000000000000000000000000000/",
    "theme.txt",
    "hotkeys.txt",
    "log.txt",
    "options.txt",
    "mlog"
]
```

But there are some issues, any tips on how to solve these would be helpful.
1. it looks like cmp-* number is dependent on latest version which is not available on install.
2. All of these files except for cmp 0s do not exist until after the app is first run. If I persist these I get folders instead. Easiest way to check is to open log.txt to read the update progress on first run, it doesn't work with persistence.
cannot persist foldit files

**Comment by jasongodev** [comment]
@schance995 What's the progress of this PR?

**Comment by jasongodev** [comment]
> I haven't played with FoldIt in a while, feel free to continue this patch.

I'm proposing to close this PR until such time it is ready or there's a lot of people requesting it in the future.

**Pull request closed without merging**